torch/nn/quantized/modules/functional_modules.py

```diff
@@ -148,7 +148,7 @@ def mul(self, x, y):
     r"""Operation equivalent to ``torch.ops.quantized.mul(Tensor, float)``"""
     def mul_scalar(self, x, y):
```

Comment:
I don't even think we need mul_scalar now - the invocation will do dispatch, no?

Reply:
ah, nevermind, it's because of different args - passing or not qscale and qbias. Maybe just hide them in the same php method and do the check on numel inside the method? (but I'm not insisting, at least add it as a comment)